osprey/processes/wps_convolution.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
                 "CASE_DIR": None,
                 "CASEID": None,
                 "CASESTR": "historical",
-                "CALENDAR": None,
+                "CALENDAR": "standard",
```

Comment:
I think it is a good idea. I don't see any constraints on doing so. How about I make a separate `templates.py` instead of including them in `utils.py`?

Reply:
I like that idea.